integreat_cms/cms/views/events/event_actions.py

```diff
@@ -59,7 +60,7 @@ def archive(
 @permission_required("cms.change_event")
 def copy(
     request: HttpRequest, event_id: int, region_slug: str, language_slug: str
-) -> HttpResponseRedirect:
+) -> HttpResponseRedirect | None:
```

Comment:
```suggestion
) -> HttpResponseRedirect:
```
This is not needed, right? Even when copying is not allowed we redirect the browser back to the view

Reply:
Oh, you're right :)